src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPDocumentMappingProvider.cs

```diff
@@ -158,6 +157,28 @@ public async override Task<TextEdit[]> RemapTextEditsAsync(Uri uri, TextEdit[] e
             return remappedEdits;
         }
 
+        public async override Task<TextEdit[]> RemapFormattedTextEditsAsync(Uri uri, TextEdit[] edits, FormattingOptions options, CancellationToken cancellationToken)
```

Comment:
This signature should also include the snapshot because we can't know if the consumer is going to do more async work and therefore need to re-check document versions

Reply:
Will add in the next PR